components/brave_extension/extension/brave_extension/webstore.ts

```diff
@@ -0,0 +1,18 @@
+const config = { attributes: true, childList: true, subtree: true }
+
+const callback = (mutationsList: MutationRecord[], observer: MutationObserver) => {
+  const button: Element = document.getElementsByClassName('g-c-R webstore-test-button-label')[0]
+
+  if (button) {
+    const text: string = button.innerHTML
+    if (text === 'Add to Chrome' || text === 'Remove from Chrome') {
```

Comment:
If this list may grow, another way to handle it would be with an array:
```js
const textToMatch: Array<string> = ['Add to Chrome', 'Remove from Chrome']
if ([textToMatch].includes(text)) {
    //..
```